### PR TITLE
tests/Added search integrations test

### PR DESCRIPTION
**Pull request by VasylLymych**
Server integration tests were added in this PR. For testing here is used package "supertest" https://www.npmjs.com/package/supertest

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -24,3 +24,5 @@ const PORT = process.env.PORT || 5000;
 
 // eslint-disable-next-line no-console
 app.listen(PORT, console.log(`Server started on port ${PORT}`));
+
+module.exports = app;
```
Comment (vyshyvanv) [inline server/server.js:28]: Please the empty line here

Comment (valentyn-arkho) [inline server/server.js]: Please add an empty line here

**Review by valentyn-arkho — CHANGES_REQUESTED** [review]
@VasylLymych , could you please resolve conflicts with the master branch?

File: package.json [hunk 2]
```diff
@@ -102,7 +103,8 @@
   },
   "jest": {
     "roots": [
-      "<rootDir>/src"
+      "<rootDir>/src",
+      "<rootDir>/server"
     ],
     "collectCoverageFrom": [
```
Comment (valentyn-arkho) [inline package.json:132]: Could you please also include `"server/**/*.js"` here? It's needed to collect coverage from all files inside of the server folder

**Review by valentyn-arkho — CHANGES_REQUESTED** [review]

**Review by valentyn-arkho — APPROVED** [review]
LGTM! 👍

**Pull request merged**